pipeline/defaults/defaults.go

```diff
@@ -37,6 +37,12 @@ func (Pipe) Run(ctx *context.Context) error {
 	if ctx.Config.ProjectName == "" {
 		ctx.Config.ProjectName = ctx.Config.Release.GitHub.Name
 	}
+	if ctx.Config.Snapcraft.Name == "" {
+		ctx.Config.Snapcraft.Name = ctx.Config.ProjectName
```

Comment:
This feels like the wrong place, don't you think?
The snapcraft pipe should take care of all the things snapcraft. This default file seems like a catch all that goes against the abstraction and modularity of releaser.

But well, this was just a comment that nobody asked about possible changes in the design, for the future. If it works now, it is good for me :D